.pre-commit-config.yaml

```diff
@@ -24,6 +24,8 @@ repos:
       - id: codespell
         name: Check for typos in code
         additional_dependencies: [tomli]
+        exclude_types:
```

Comment:
what is it complaining on?

Reply:
\n or that we have a lot of typos in docstrings? or that it doesnt make sense because there are non-words that we dotn want to fix?

Reply:
Complained about `nin` not being correct, combination of `\n` and `in` without space